### PR TITLE
Create connect-redis definitions

### PR DESCRIPTION
**Pull request by Xstoudi**

### CODE REVIEW

**Comment by Xstoudi** [comment]
How can I inject dependencies ?

**Comment by vvakame** [comment]
please use reference comment.
like this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/redis/redis.d.ts#L8

**Comment by Xstoudi** [comment]
Up ?

File: connect-redis/connect-redis-tests.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/// <reference path="./connect-redis/connect-redis.d.ts" />
```
Comment (vvakame) [inline connect-redis/connect-redis-tests.ts]: `./connect-redis/connect-redis.d.ts` -> `./connect-redis.d.ts` (relative path!

**Pull request merged**